glossary/glossary.md

```diff
@@ -4,85 +4,85 @@
 :   Software that is no longer being maintained.
 
 **absolute error**<a id="absolute_error"></a>
-:   The absolute value of the difference between the observed and the correct value. Absolute error is usually less useful than [relative error](#relative_error).
+:   The absolute value of the difference between the observed and the correct value. Absolute error is usually less useful than \gref{relative error}{relative\_error}.
 
 **absolute import**<a id="absolute_import"></a>
 :   In Python, an import that specifies the full location of the file to be imported.
 
 **absolute path**<a id="absolute_path"></a>
-:   A path that points to the same location in the filesystem regardless of where it's evaluated. An absolute path is the equivalent of latitude and longitude in geography. See also: [relative path](#relative_path)
+:   A path that points to the same location in the \gref{filesystem}{filesystem} regardless of where it is evaluated. An absolute path is the equivalent of latitude and longitude in geography. See also: [relative path](#relative_path)
 
 **actual result (of test)**<a id="actual_result"></a>
-:   The value generated by running code in a test. If this matches the [expected_result](#expected_result), the test [passes](#pass_test); if the two are different, the test [fails](#fail_test).
+:   The value generated by running code in a test. If this matches the \gref{expected result}{expected\_result}, the test \gref{passes}{pass\_test}; if the two are different, the test \gref{fails}{fail\_test}.
```

Comment:
Do the backslashes need to be used? 

In the html version of the book, `{pass\_test}` is a hyperlink to nowhere, whereas `{pass_test}` links correctly.